assets/css/admin-custom.css

```diff
@@ -117,3 +117,66 @@ div.question_boolean_fields { margin-bottom: 10px; }
 .sensei-dismissible-link {
     text-decoration: none;
 }
+
+/**
+ * Sensei custom navigation
+ */
+.sensei-custom-navigation {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	gap: 20px;
+	margin: 20px auto;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__main {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+	justify-content: flex-start;
+	gap: 40px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__main .sensei-custom-navigation__main__title h1 {
```

Comment:
Is there a practical need for this selector to be so specific? Would `.sensei-custom-navigation__main__title h1` work? Wondering the same thing for the other selectors in this file.

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4672/commits/08e1fa3d2e76062ef6abe8c958ba000f3131b54e